.github/workflows/cicd.yml

```diff
@@ -0,0 +1,27 @@
+name: Continuous Integration
+
+on:
+  push:
+    branches: 
+      - main
+      - develop
+  pull_request:
+    branches: 
+      - main
+      - develop
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    # Checkout the repository for processing
+    - name: Checkout Repository
+      uses: actions/checkout@v2
+
+    # Test the output files to confirm they're both conformant to the CASE Ontology
+    - name: CASE Validation
+      uses: kchason/case-validation-action@v2.3
+      with:
+        case-path: ./json/
+        case-version: "case-1.0.0"
```

Comment:
All of the IRI prefixes will need to be fixed before this line can fruitfully assign `case-1.0.0`.

@fabrizio-turchi , would you rather the validation workflow start with `case-1.0.0`, or instead start with (IIRC) `case-0.6.0`?

Reply:
I suggest the CASE and UCO IRI prefixes be revised to use the CASE/UCO 1.0.0 form, so this repo's maintenance work focus on the 1.0.0 release instead of conforming to 6 months ago and *then* doing a lift to get to CASE 1.0.0.

Reply:
Oh, some of these were developed against CASE 0.4.0, which is pre-SHACL.

Disregard my alternative suggestion - please jump straight to validation against 1.0.0.

Reply:
I updated URI prefixes and the uco-core:specVersion property to the "CASE 1.0.0" value

Reply:
I see the update.  Thanks!